docs/codeReview.md

```diff
@@ -57,14 +71,4 @@ A PR advances from self-review to reviewer-review and finally maintainer review
     - resolve if you followed the suggestion
     - reply if you didn't
 - If you need to clarify parts of the code, check if it can be done by adding comments or improve naming of variables/functions/classes
-- When you replied or resolved all comments, move issue back to reviewer- or maintainer-review
-
-## One Review Required
-
-If the developer creating the PR feels that maintainer review isn't necessary, they should add a label: “One Review Required”. If the Reviewer thinks it's not safe to have just one review, it's their responsiblity to add a comment saying that Maintainer review is required to merge the PR.
-
-Examples of PRs that could be safe to merge that way
-
-- Minor changes, e.g. [remove octokit](https://github.com/ParabolInc/parabol/pull/6479)
-- Small, non-architectural changes that happen behind a feature flag, e.g. [show GitLab issues in the Estimate phase](https://github.com/ParabolInc/parabol/pull/6355). It's the PR author's responsibility to keep track of the scenarios that need testing before removing the feature flag.
-- Work that is related to new features which are hidden after a feature flag and do not touch existing functionality (any work related to the architecture of the new feature is an exception and should go through the full review process) e.g [added end team prompt mutation](https://github.com/ParabolInc/parabol/pull/6250)
+- When you replied or resolved all comments, move the PR back to reviewer- or maintainer-review
```

Comment:
@mattkrick are you still following the reviewer-review / maintainer-review columns or could we simplify this to just "review" and just follow `is:pr is:open user-review-requested:@me` search?

Reply:
that works for me! this trimester my exco responsibilities are preventing me from doing as much in the codebase as I'd like so I want to remove myself as a blocker as much as I can

Reply:
Changed to just "back to review"